docs/source/https.md

```diff
@@ -27,6 +23,10 @@ up **manually** with your own key and certificate.
     ```python
     c.Proxy.traefik_acme_server="https://acme-v02.api.letsencrypt.org/directory"
     ```
+    * Set the port to be used by Traefik for the Acme HTTP challenge:
+    ```python
+    c.Proxy.traefik_acme_challenge_port="https://acme-v02.api.letsencrypt.org/directory"
```

Comment:
Hmmm, Traefik does indeed need to listen to HTTP based traffic, so that it can receive traffic from an ACME server sending out a HTTP-01 request to port 80. I'm not sure how the configuration is setup etc, but perhaps you can rephrase this to indicate that its the port where traefik will be _listening_ to a HTTP-01 challenge request from an ACME server, which only sends out request to port 80 (except for Pebble - the ACME testing server).